src/server.py

```diff
@@ -1,16 +1,32 @@
 import asyncio
 import asyncpg
+import secrets
+import string
 import random
+import time
+import json
+from datetime import datetime, timedelta
 from contextlib import asynccontextmanager
 from fastapi import FastAPI, HTTPException
 from fastapi.staticfiles import StaticFiles
 from deck import Deck
 
 SUITS = ["C", "D", "H", "S"]
-RANKS = ["2", "3", "4", "5", "6", "7", "8", "9", "0", "J", "Q", "K", "A"]
+RANKS = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
 
 app = FastAPI()
 
+# Dictionaries to store client IDs and associated objects
+deck_objects: dict[str, Deck] = {}
+expiration_times: dict[str, float] = {}
```

Comment:
Why do we need expiration? It's probably okay to not track that, and just keep things around for the time being.

It's not ideal, but you can't really guarantee that someone won't walk away from their browser for a long time and come back to play (unless the expiration on the server does cause a change in the UI, which would be tricky).

Reply:
@semmypurewal I was initially thinking of this as authorization, but I now agree that 24 hours is a bit strict for such a small amount of data being stored.